app/views/core_induction_programmes/years/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<%= govuk_breadcrumbs breadcrumbs: course_year_breadcrumbs(current_user, @course_year) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Your module materials</h1>
+
+    <% if @cip.course_year_two_id %>
+      <%= render SubnavComponent.new do |component| %>
+        <%= component.nav_item(path: year_path(@cip.course_year_one_id)) do %>
+          Year 1
+        <% end %>
+        <%= component.nav_item(path: year_path(@cip.course_year_two_id)) do %>
+          Year 2
+        <% end %>
+      <% end %>
+    <% end %>
+
+    <% if current_user&.admin? || policy(@course_year).edit? %>
```

Comment:
Why would we need both an admin check and policy check?

Reply:
This code was moved from c_i_p/show.html.erb - I didn't write it

Reply:
oh wait i totally did write this

i was just combining the two existing if statements to see if there should be a button group - i didn't really think about it that hard